django/thunderstore/api/cyberstorm/views/team.py

```diff
@@ -39,6 +46,40 @@ def check_permissions(self, request: Request) -> None:
             raise PermissionDenied()
 
 
+class CyberstormTeamCreateRequestSerialiazer(serializers.Serializer):
+    name = serializers.CharField(
+        max_length=Team._meta.get_field("name").max_length,
+        validators=Team._meta.get_field("name").validators,
+    )
+
+
+class CyberstormTeamCreateResponseSerialiazer(serializers.Serializer):
+    name = serializers.CharField()
+
+
+class TeamCreateAPIView(APIView):
+    @conditional_swagger_auto_schema(
+        request_body=CyberstormTeamCreateRequestSerialiazer,
+        responses={200: CyberstormTeamCreateResponseSerialiazer},
+        operation_id="cyberstorm.teams.create",
+        tags=["cyberstorm"],
+    )
+    def post(self, request, format=None):
+        serializer = CyberstormTeamCreateRequestSerialiazer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+
+        form = CreateTeamForm(
+            user=request.user,
+            data=request.data,
```

Comment:
Neve ruse request.data directly when you use serializers to validate it; use `serializer.validated_data`